SingularityBase/src/main/java/com/hubspot/singularity/SingularityTaskWebhook.java

```diff
@@ -14,10 +15,12 @@ public SingularityTaskWebhook(@JsonProperty("task") SingularityTask task, @JsonP
     this.taskUpdate = taskUpdate;
   }
 
+  @ApiModelProperty(required=false, value="The task this webhook refers to.")
```

Comment:
Due to the fact that this is not a model returned by a specific endpoint, I don't think swagger will pick it up without an `@ApiModel` annotation on the class as a whole. You can test this out running something like `mvn -Pbuild-swagger-documentation -DskipTests=true -B -q -fae install` to generate the documentation and check if the model appears

Reply:
Hmm good catch. Even with the `@ApiModel` annotation it doesn't appear though. Google's not telling me much about this. I've compared against the other classes annotated with this in Singularity too and it seems okay. Any ideas?

Reply:
Update - added the link to github instead of the docs link since swagger still isn't building this even when marked with the annotation.